portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SegmentsCreationWebBehavior.testcase

```diff
@@ -1373,4 +1373,46 @@ definition {
 		ACUtils.viewNameList(nameList = "ac ac");
 	}
 
+	@description = "Bug: LRAC-8233 | Automation ID: LRAC-11537 | Test Summary: Check web behavior shows which data source data came from"
+	@priority = "3"
+	test CheckWebBehaviorSegmentShowsWhichDataSourceDataCameFrom {
+		ACUtils.createBlogsAndAddToPage();
+
+		var propertyName = ACDXPSettings.connectDXPtoAnalyticsCloud(siteName = "Site Name");
+
+		var dataSourceName = StringUtil.extractFirst("${propertyName}", "Combined");
```

Comment:
I think you could change it to use `${assignedPropertyName}`. What do you think?